core/pom.xml

```diff
@@ -12,6 +12,7 @@
 
   <artifactId>streamx-cli-core</artifactId>
 
+  <name>StreamX Cli : CORE</name>
```

Comment:
My bad. I meant not uppercase but capitalised...

So here: `StreamX Cli : Core`